ibm_catalog.json

```diff
@@ -170,7 +170,27 @@
                   ]
                 },
                 {
-                  "key": "elasticsearch_version"
+                  "key": "elasticsearch_version",
+                  "required": true,
+                  "default_value": "",
+                  "options": [
+                    {
+                      "displayname": "v8.12 (preferred)",
+                      "value": "8.12"
+                    },
+                    {
+                      "displayname": "v8.10",
+                      "value": "8.10"
+                    },
+                    {
+                      "displayname": "v8.7",
+                      "value": "8.7"
+                    },
+                    {
+                      "displayname": "v7.17 (deprecated)",
```

Comment:
we should not have support for a deprecated version

Reply:
sure, will remove it.